examples/chart/teleport-cluster/templates/auth/deployment.yaml

```diff
@@ -266,7 +266,7 @@ spec:
 {{- if $auth.enterprise }}
       - name: license
         secret:
-          secretName: "license"
+          secretName: {{ $auth.licenseSecretName | default "license" | quote }}
```

Comment:
nit: can you move the default `licenseSecretName` into the values instead of the template? We are not generating reference yet for `teleport-cluster` but this will allow us to parse and display the default value in the ref rather than maintaining it manually.

Reply:
Yes, of course! I just pushed the update.